app/models/financialStatement/PsaFSDetail.scala

```diff
@@ -41,8 +42,20 @@ case class PsaFSDetail(chargeReference: String,
                        periodEndDate: LocalDate,
                        pstr: String,
                        sourceChargeRefForInterest: Option[String],
+                       sourceChargeInfo: Option[SourceChargeInfo] = None,
                        documentLineItemDetails: Seq[DocumentLineItemDetail])
 
 object PsaFSDetail {
   implicit val formats: Format[PsaFSDetail] = Json.format[PsaFSDetail]
 }
+
+case class SourceChargeInfo(
```

Comment:
Could this be changed to PSASourceChargeInfo so it doesn't clash with the scheme version? Same in back end. Will rename the scheme one